telemetry/metrics.yaml

```diff
@@ -1,24 +1,250 @@
----
-# Schema
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
 
-page:
-  load:
+identifiers:
+  fxa_account_id:
```

Comment:
Did we try to attach the FxA account ID of MoSo users to any of the sample events?

Reply:
@kirill-demtchouk Our clients as of today, will not have the FxA id.

Reply:
Hopefully we can change that in the future, but right now they only know about mastodon data.

Reply:
Ah, right, good point...once you're authed by the FxA flow then you probably know nothing about it. Let's not worry about this for now.